cf/query.py

```diff
@@ -1671,9 +1709,20 @@ def wi(value0, value1, units=None, attr=None):
     True
     >>> q.evaluate(4)
     False
+    >>> q.evaluate(5)
+    True
+    >>> q.evaluate(5, open_lower=True)
```

Comment:
This example doesn't look right - it's `cf.wi` that has the `open_lower` keyword, not `evaluate`.